indexTest.js

```diff
@@ -1,3 +1,2 @@
-const testsContext = require.context('.', true, /\/(src|platform|\.\/example)\/.*Spec.js$/);
-
+const testsContext = require.context('.', true, /^\.\/(src|example)\/.*Spec.js$/);
```

Comment:
Adjusted regex so that we don't pick up:
`./node_modules/fooPackage/src/FooSpec.js`
`./node_modules/fooPackage/example/FooSpec.js`
but we do pick up:
`./example/eventGenerator/pluginSpec.js`

As the `example` directory is often released to users, it seems like it should be covered by tests